react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class hotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleClick = this.handleClick.bind(this);
+    this.i = 0;
```

Comment:
In React we want to avoid sticking extra data on `this`. Put the number of clicks in your `state`.